test/site/get-site-status-by-exporter-name.api-test.ts

```diff
@@ -105,7 +105,7 @@ describe('getSiteStatusByExporterName', () => {
     expect(body).toStrictEqual({ message: 'Not found', statusCode: 404 });
   });
 
-  it('returns a 400 with message containing "property does not exist" if query property does not exist', async () => {
+  it("returns a 400 because query parameter is missing, but incorrect query parameter doesn't trigger error", async () => {
```

Comment:
Any reason we are repurposing this test? I'd expect the old test still to pass?

Reply:
Luke highlighted that ValidationPipe configuration in TFS is different, so I removed forbidNonWhitelisted: true. This change broke this GET site test.

Reply:
I think we should either just delete this test or make it "if you use the _correct_ query parameter _and_ an unexpected one as well then it still succeeds/you get a 200 response" - otherwise it's a bit confusing what exactly this test is for :)

Reply:
Changed test to "if you use the correct query parameter and an unexpected one as well, then it still succeeds/you get a 202 response". 202 "Provisioning" is default status in get site generator.